src/command.c

```diff
@@ -897,8 +909,9 @@ static int dv_convert_ipu_v0(struct device *dev, struct dmp_cmb *cmb,
 	uint32_t rd_base_addr = 0, rd_buf_size;
 	uint32_t wr_base_addr = 0, wr_buf_size;
 	int ret;
-	
-	cmb_node = list_first_entry(&cmb->cmb_list, struct dmp_cmb_list_entry, list_node);
+
+	cmb_node = list_first_entry(&cmb->cmb_list,
+			struct dmp_cmb_list_entry, list_node);
```

Comment:
misalignment, either break line just after the bracket or align next line just after the bracket